internal/provider/resource_rediscloud_subscription_peering.go

```diff
@@ -48,35 +49,56 @@ func resourceRedisCloudSubscriptionPeering() *schema.Resource {
 				ValidateDiagFunc: validateDiagFunc(validation.StringMatch(regexp.MustCompile("^\\d+$"), "must be a number")),
 				ForceNew:         true,
 			},
+			"provider_name": {
+				Type:             schema.TypeString,
```

Comment:
Missing a `Description` here

Reply:
This has been added and matches the docs